content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,81 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+### Features
+
+#### Replicate data remotely
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+This feature helps you to collect data locally from multiple InfluxDB OSS edge deployments, analyze and aggregate the data, and then send the data to one or more buckets on a remote InfluxDB Cloud instance. If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+- https://github.com/influxdata/influxdb/commit/5a1e375): Build: allow to build on FreeBSD
+
+#### Task metadata
+
+- (https://github.com/influxdata/influxdb/commit/504f0e4): Passing `type=basic` returns task metadata without query text.
+
+#### Metrics
+
+(https://github.com/influxdata/influxdb/commit/3a81166): Added metrics collection for replications
+(https://github.com/influxdata/influxdb/commit/dece95d): TSM compaction metrics via prometheus
+
+- Add the following metrics when data fails to be added to the replication queue:
+  `PointsFailedToQueue`
+  `BytesFailedToQueue`
+
+- Add the following metrics for writes, to indicate the status:
+	- `pointsWriteRequested`
+	- `pointsWriteOk`
+	- `pointsWriteDropped`
+	- `pointsWriteErr`
+	- `timeout`
```

Comment:
@jstirnaman lines 40-49 are the metrics we chatted about this morning that you guys thought may be covered by the rejected points content...to be sure, anything to clarify or add in context?

Would it make sense to send folks here?:

https://docs.influxdata.com/influxdb/v2.1/write-data/troubleshoot/#troubleshoot-rejected-points
Or here:
https://docs.influxdata.com/influxdb/v2.1/api/#tag/Write

Reply:
Metrics doc will be at https://docs.influxdata.com/influxdb/v2.1/reference/internals/metrics (I hadn't merged it yet). We'll need to add these new metrics there. I suppose we should also mention them in https://docs.influxdata.com/influxdb/v2.1/write-data/troubleshoot/#troubleshoot-rejected-points like you suggested.